### DISCUSSION TITLE
New transaction - current time

### DISCUSSION THREAD

**Discussion by thehijacker** [firefly-iii/firefly-iii#8318] [General]
When I create new transaction the time in date/time field is always set to 00:00. Can it be set to current time?

![image](https://github.com/firefly-iii/firefly-iii/assets/13857043/f7b68ddc-6867-4902-83bc-2b1219a4fc7a)

Thank you.

**Comment by JC5** [comment]
Sure, good idea!

**Comment by kmarnold24** [comment]
How about a setting for a default time? I like 00;00 since time on a transaction isn't a huge deal for me.  Or just mask time all together

**Comment by thehijacker** [comment]
Sorting of transaction is based on transaction time. If you wish to have them in same order as on bank transaction list, it is crucial to have correct date and time. I also use Watefly as my Android Firefly GUI in which I input transaction. Waterfly will always also date and time.

For me currect setting to also add time is very crucial.

**Comment by JC5** [comment]
There will not be a setting. For new transactions, the form will default to now, ie. 13:48. For existing transactions, the time of the transaction will be used or 00:00 if it's not set.
